salt/modules/cron.py

```diff
@@ -91,7 +91,7 @@ def _cron_matched(cron, cmd, identifier=None):
                 cid = eidentifier
             id_matched = eidentifier == cid
     if (
-        ((id_matched is None) and cmd == cron.get('cmd', None))
+        ((id_matched) and cmd == cron.get('cmd', None))
```

Comment:
This seems redundant to me. We have an `or id_matched` down below on L95.

@kiorky you refactored this a little while ago. Can you comment here?

Reply:
@freelingchang Did you see my comment about this possibly being redundent?